src/util/obbt.jl

```diff
@@ -191,23 +193,30 @@ function run_obbt_opf(data::Dict{String,Any}, solver;
         for bus in buses
             (vm_ub[bus] - vm_lb[bus] < min_bound_width) && (continue)
             
+            start_time = time()
             # vm lower bound solve 
             lb = NaN 
             @objective(model_bt.model, Min, vm[bus])
             result_bt = solve_generic_model(model_bt, solver)
-            if (result_bt["status"] == :LocalOptimal || result_bt["status"] == :Optimal)
+            if (result_bt["status"] == :LocalOptimal || result_bt["status"] == :Optimal || result_bt["status"] == :SubOptimal)
```

Comment:
In my experince, SubOptimal status from Gurobi means that the solution and objective value are not correct.  This should be treated more like `:Infeasible` than `:Optimal`.

Reply:
Stop by ANSI conference room if you would like to discuss.